pages/common/vim.md

```diff
@@ -14,6 +14,28 @@
 
 `view {{file}}`
 
-- Exit vim:
+- Save and Exit vim:
 
-`[Esc] (to switch to normal mode), then :q`
+`<Esc>:wq<Cr>`
+
+- Enter insert mode:
```

Comment:
I'd further clarify this as "insert (normal editing) mode", or something to that effect.